src/app/components/article/SavedSearchArticles.js

```diff
@@ -0,0 +1,110 @@
+
+const SavedSearchArticles = ({ routeParams }) => (
+  <ErrorBoundary component="SavedSearch">
+    <QueryRenderer
+      environment={Relay.Store}
+      query={graphql`
+        query SavedSearchArticleshQuery($id: ID!) {
+          saved_search(id: $id) {
+            id
+            dbid
+            title
+            filters
+            list_type
+            medias_count: items_count
+            team {
+              id
+              slug
+              name
+              permissions
+            }
+          }
+        }
+      `}
+      render={({ error, props }) => {
+        if (!error && props) {
+          // Weird Relay error happens here if "filters" is a JSON object instead of a JSON string...
+          // "Uncaught TypeError: Cannot assign to read only property '<filter name>' of object '#<Object>'"
+          const defaultQuery = safelyParseJSON(props.saved_search.filters, {});
+          const query = routeParams.query ? safelyParseJSON(routeParams.query, {}) : defaultQuery;
+
+          return (
+            <Articles
+              defaultFilters={defaultQuery}
+              filterOptions={['users', 'tags', 'range', 'verification_status', 'language_filter', 'published_by', 'channels']}
+              icon={<ListIcon />}
+              listActions={
+                <ProjectActions
+                  deleteMessage={
+                    <FormattedHTMLMessage
+                      defaultMessage="Are you sure? This is shared among all users of <strong>{teamName}</strong>. After deleting it, no user will be able to access it."
+                      description="A message that appears when a user tries to delete a list, warning them that it will affect other users in their workspace."
+                      id="savedSearchArticles.deleteMessage"
+                      tagName="p"
+                      values={{
+                        teamName: props.saved_search?.team ? props.saved_search.team.name : '',
+                      }}
+                    />
+                  }
+                  deleteMutation={graphql`
+                    mutation SavedSearchArticlesDestroySavedSearchMutation($input: DestroySavedSearchInput!) {
+                      destroySavedSearch(input: $input) {
+                        deletedId
+                        team {
+                          id
+                        }
+                      }
+                    }
+                  `}
+                  object={props.saved_search}
+                  updateMutation={graphql`
+                    mutation SavedSearchArticlesUpdateSavedSearchMutation($input: UpdateSavedSearchInput!) {
+                      updateSavedSearch(input: $input) {
+                        saved_search {
+                          id
+                          title
+                          medias_count: items_count
+                        }
+                      }
+                    }
+                  `}
+                />
+              }
```

Comment:
This looks like a good chunk of duplication with the one in `SavedSearch.js`. I suggest moving all the common code into `ProjectActions` and doing any specialization logic in there. It would just take a "type" prop to know which kind of list: articles or media.

This would also make this component and `SavedSearch` easier to read.

Reply:
Done and rename `ProjectActions` to be `SavedSearchActions`